src/analysis/bounds.rs

```diff
@@ -177,6 +177,9 @@ impl Bounds {
             Type::Fundamental(Fundamental::Filename) => Some(AsRef(None)),
             Type::Fundamental(Fundamental::OsString)=> Some(AsRef(None)),
             Type::Fundamental(Fundamental::Utf8) if *nullable => Some(Into(Some('_'), None)),
+            // FIXME: Here we need to access analysis::Object::final_type to know if we have a
+            // final type or not. How?
+            // Final types should not use IsA<_>
```

Comment:
@EPashkin do you have an idea how to do this here? This also seems a bit problematic as it requires analysis for the type to be done already, but this function here for the bounds could be called before (or not?).

Reply:
We can add final_type to library::Class and set it in post-processing.

Reply:
Postprocessing happens after parsing the config and .girs, and before analysis? So that I can basically do the same there as I'm currently doing in analysis? That would work